app.js

```diff
@@ -1,10 +1,53 @@
 const WebSocket = require('ws');
 require('dotenv').load();
-var exp = require('./index.js');
 
+// when socket opens, notify channel
+function onOpen (soc) {
+    console.log('socket opened');
+    var msg = {
+        "id": 1,
+        "type": "message",
+        "channel": "C1BBWJ7PF",
```

Comment:
Future work is to look at the object that comes back from the `rt.start` call and read the channel IDs from that, so that you can put friendly channel names in here.